lib/rules/no-multi-spaces.js

```diff
@@ -93,7 +93,7 @@ module.exports = {
                 const sourceCode = context.getSourceCode(),
                     source = sourceCode.getText(),
                     allComments = sourceCode.getAllComments(),
-                    pattern = /[^\n\r\u2028\u2029\t ].? {2,}/g;  // note: repeating space
+                    pattern = /[^\s].? {2,}/g;  // note: repeating space
```

Comment:
Hmm, looks like you're right. I'll switch it back.

The effect of this is that the following code would not be reported:

```js
foo\f\f  + 1
```